build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -81,8 +81,9 @@ CONFIG_ARCH_ARMV8M_FAMILY=y
 CONFIG_ARCH_FAMILY="armv8-m"
 CONFIG_ARCH_HAVE_CMNVECTOR=y
 CONFIG_ARM_CMNVECTOR=y
-# CONFIG_ARCH_HAVE_FPU is not set
+CONFIG_ARCH_HAVE_FPU=y
```

Comment:
Is there any specific reason to use FPU?

Reply:
The amebad chip has a fpu in the hardware. Due to this, when an exception happens, the hardware saves the fpu registers on the stack by default. But if we do not set the FPU config in software, then this somehow affects the default stacking mechanism during exception handling. This is resulting in a corruption of PC register value when returning from the rdp demo test.  I think whenever, a FPU hardware is present, it is mandatory to set this config for proper operation.

Reply:
Could you enable this in all of defconfigs for realtek board?

Reply:
Done.